src/bin/write/mod_csv.c

```diff
@@ -1,7 +1,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <fcntl.h>
-#include <unistd.h>
+#if !defined(_MSC_VER)
+#   include <unistd.h>
+#endif
```

Comment:
I am guessing that most inclusions of `unistd.h` are no longer needed even on Unix, but I can clean them up later.